### PR TITLE
add isLoggedin to props

### PR DESCRIPTION
**Pull request by erickj04**
- add isLoggedin to props in timeline

### CODE REVIEW

File: yarn.lock
Comment (svolterra) [inline yarn.lock:1]: Please do not include this file in your PR :)

File: src/pages/Timeline/Timeline.tsx [hunk 1]
```diff
@@ -24,6 +26,10 @@ const Timeline: React.FC<TimelineProps> = (props) => {
     contributors: Array<string>;
     updatedTime: string;
   }
+
+  const isLoggedIn = useAppSelector(selectIsLoggedIn);
+  console.log(isLoggedIn);
```
Comment (svolterra) [inline src/pages/Timeline/Timeline.tsx:31]: Please delete this `console.log(isLoggedIn)` line 😄

**Review by svolterra — COMMENTED** [review]
Looks good to me! 🥳 👏 Just left a couple of comments regarding the PR! 😄 

---

Small note just in case:

Once we figure out why the logging in functionality isn't working, we may decide to access `isLoggedIn` directly through the `TimelineCommitBlock.tsx` file instead of passing it in as a prop from the `Timeline.tsx` file.

File: yarn.lock
Comment (erickj04) [reply yarn.lock:1]: okay :)

**Comment by erickj04** [comment]
done!

**Comment by ElisaHu1** [comment]
image
a subtle ui bug introduced here

**Comment by ElisaHu1** [comment]
image
looks good, seems the bug is only present when user not login

**Comment by erickj04** [comment]
oh yeah...nice catch! I've just fixed it

**Review by ElisaHu1 — APPROVED** [review]
great work! you can merge this now!

**Pull request merged**